### ISSUE TITLE
Bug: Insufficient Details in Error Messages

### ISSUE THREAD

**Issue by ravensorb** [structr/structr#482]
I am working with the REST and in the latest 2.1 builds there is no useful error information being returned when POSTing data

Example Request
```
127.0.0.1 - - 18/Nov/2016:17:48:19 +0000 "POST /structr/rest/location HTTP/1.1" 422 98
POST /structr/rest/venue HTTP/1.1
Cookie: JSESSIONID=[Session]
Accept: */*
Connection: keep-alive
Host: 127.0.0.1:8082
Accept-Encoding: gzip, deflate
Content-Length: 533
Content-Type: application/json; charset=UTF-8

{
  "name": "Revolution Place",
  "phoneNumber": "",
  "faxNumber": "",
  "customerServiceEmail": "",
  "salesEmail": "",
  "capacity": 3228,
  "dateOpened": "1995-01-01T00:00:00+0000",
  "address": {
    "line1": "",
    "line2": "",
    "city": "Grand Prairie",
    "region": "TX",
    "country": "US",
    "postalCode": ""
  },
  "events": [],
  "source": {
    "sourceName": "google",
  }
}
```

Example Response
```
HTTP/1.1 422 NA
Date: Fri, 18 Nov 2016 17:48:18 GMT
Content-Type: application/json; charset=utf-8;

{
  "code": 422,
  "message": "Unable to commit transaction, validation failed",
  "errors": []
}
```

What am I supposed to do with that :)

Note: /var/log/structr-ui.log has NO information at all when this error occurs.

**Comment by cmorgner** [comment]
Hi, there was indeed a bug in one of the snapshots, which version are you using? The bug is fixed in the current master, but the fix might not be deployed as a snapshot yet.

Best,
Christian



Am 18.11.2016 um 18:54 schrieb Shawn:

**Comment by ravensorb** [comment]
Ok, good to know it was fixed :)

I was using  structr-ui-2.1.0-20161112.121104-20 and am just moving up to  structr-ui-2.1.0-20161114.212029-21

**Comment by cmorgner** [comment]
The fix is in commit 5cde3423a7be7d3fbb8d2be7da2d8439b581f22d from Mon Nov 14 22:17:47 2016 +0100


Am 18.11.2016 um 18:57 schrieb Shawn:
